### ISSUE TITLE
seasons for winter-sports (skiing)

### ISSUE THREAD

**Issue by Shura0** [SamR1/FitTrackee#133]
There would be nice to have different seasons for summer and winter-sports.
Usually, skiing season starts in November and ends in April, so with current year statistic the season splits into two years (2021 and 2022 for now) and it is not convenient. It would be great to have statistic like this:
![изображение](https://user-images.githubusercontent.com/2156669/148396935-242ef502-7d22-4695-adfb-ea23d1e19600.png)
Bike and run on top, skiing in bottom (or it can be on top too, but between running seasons)

**Comment by SamR1** [comment]
Hi,

something like this?

![screenshot](https://user-images.githubusercontent.com/3525167/148639422-ce1e23d8-f2bd-47d5-b139-387a5b149c84.png)

with the same workouts displayed as follows:

![example](https://user-images.githubusercontent.com/3525167/148639432-f83d7d6a-d31d-4db7-9c70-c0c2427aecf4.png)  
(_it's just a screenshot modified with Gimp_ :) )

**Comment by Shura0** [comment]
Yes, maybe like this. But "summer' label for bike it's too confusing, bike (and run) season should be a whole year.

**Comment by SamR1** [comment]
Sorry, I misunderstood I think. In my mind, these were just 6 month periods (all workouts matching the dates will be displayed in the corresponding time period regardless of the sport). For instance a bike workout in December will be displayed in "winter" period.

The sessions would be grouped by sport according to the season? For a given year, all ski or winter sports sessions will be displayed in the same category, regardless of the month. And other sports in another category?

**Comment by Shura0** [comment]
Oh, now I got your point. You want to add one more point of statistic view - season.
Well, it will be OK.

My idea was to have separate view for winter sports in 'year' graph, when summer sports are as is, have year statistic, but winter-sports (a special category of sports) have seasoned statistic. Seasons are Oct - May

**Comment by SamR1** [comment]
OK.  
Adding "season" view and selecting only winter sports could be OK (but only one chart would be displayed), before working back on Statistics view which is quite limited for now.
